editor/project_converter_3_to_4.cpp

```diff
@@ -120,7 +120,7 @@ static const char *enum_renames[][2] = {
 	{ "JOINT_PIN", "JOINT_TYPE_PIN" }, // PhysicsServer2D
 	{ "JOINT_SLIDER", "JOINT_TYPE_SLIDER" }, // PhysicsServer3D
 	{ "KEY_CONTROL", "KEY_CTRL" }, // Globals
-	{ "LOOP_PING_PONG", "LOOP_PINGPONG" }, //AudioStreamSample
+	{ "LOOP_PING_PONG", "LOOP_PINGPONG" }, //AudioStreamPCM
```

Comment:
I am not familiar with the project converter, but is there actually some code that converts "AudioStreamSample" to "AudioStreamPCM"? I just see a comment change here.

Reply:
I wasn't aware of the class renames, I added that now.